distribution/lib/Standard/Examples/0.0.0-dev/src/Main.enso

```diff
@@ -164,8 +164,8 @@ http_client : Http
 http_client = Http.new (timeout = 30.seconds)
 
 ## A basic URI for examples.
-uri : Uri
-uri = Enso_Uri.parse "http://user:pass@example.com/foo/bar?key=val"
+uri : URI
+uri = Enso_URI.parse "http://user:pass@example.com/foo/bar?key=val"
```

Comment:
I think the type should then be `Enso_URI` too, due to the rename (if the rename is kept).